.github/workflows/dapp.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   setup-local-hedera:
     name: Dapp Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 35
+    timeout-minutes: 180
```

Comment:
Yeah, I agree that we should aim to bring down this time, I remember this Test used to take around ~22 mins, not sure what happened for it to increase that much up to 35, since must of time is failing due to time-out, but the test is still going and running. So I think we should jump it to 45 min and open a ticket to see if there is a way to optimize it.

Reply:
Done